syft/workers/base.py

```diff
@@ -226,6 +228,24 @@ def _recv_msg(self, message: bin):
         """
         raise NotImplementedError  # pragma: no cover
 
+    def register_obj(self, obj):
+        self.object_store.register_obj(self, obj)
+
+    def clear_objects(self, return_self: bool = True):
+        """Removes all objects from the object storage.
+
+        Note: the "return self" statement is kept in order to avoid modifying the code shown in the udacity course.
+
+        Args:
+            return_self: flag, whether to return self as return value
+
+        Returns:
+            self, if return_self if True, else None
+
+        """
+        self.object_store.clear_objects()
+        return self if return_self else None
```

Comment:
I'm not sure we'll ever want to return a `Worker` when calling a function called `clear_objects`, right?
Maybe we could return `self.object_store`?

Reply:
Yeah, this is not great, but it's required to maintain compatibility with the Udacity course. I'll add a comment to that effect.